### PR TITLE
OCPBUGS-41924, OCPBUGS-43540: Update and enable shipwright e2e test on CI

### PR DESCRIPTION
**Pull request by The-Anton**
**Fix**:
[OCPBUGS-41924](https://issues.redhat.com/browse/OCPBUGS-41924)
[OCPBUGS-39573](https://issues.redhat.com/browse/OCPBUGS-39573)
**Description**

1. Enable Shipwright e2e tests.
2. Updates Shipwright Operator subscription to latest release
3. Disable `SWB-01-TC06 & SWB-01-TC08` tests

### CODE REVIEW

**Comment by vikram-raj** [comment]
/retitle OCPBUGS-41924: Update Shipwright Operator Subscription to startingCSV v0.13.0

File: frontend/integration-tests/test-cypress.sh [hunk 1]
```diff
@@ -90,10 +90,10 @@ if [ -n "${nightly-}" ] && [ -z "${pkg-}" ]; then
...
 
 if [ -n "${headless-}" ] && [ -z "${pkg-}" ]; then
-  yarn run test-cypress-console-headless
-  yarn run test-cypress-dev-console-headless
-  yarn run test-cypress-olm-headless
-  yarn run test-cypress-helm-headless
+  # yarn run test-cypress-console-headless
+  # yarn run test-cypress-dev-console-headless
+  # yarn run test-cypress-olm-headless
+  # yarn run test-cypress-helm-headless
```
Comment (invincibleJai) [inline frontend/integration-tests/test-cypress.sh:96]: why are all of these tests disabled?

Comment (The-Anton) [reply frontend/integration-tests/test-cypress.sh]: The test were disabled temporarily due to investigate shipwright tests in isolation on CI.

**Review by logonoff — COMMENTED** [review]
`shipwright-ci` runs green but I am getting errors for other features. Do these need to be fixed as well?

shipwright-ci

create-shipwright-build-form

**Comment by The-Anton** [comment]
Thanks, @logonoff for testing it, Currently, the PR only aims for the `shipwright-ci` test.

**Pull request merged**